src/scancode/cli.py

```diff
@@ -255,26 +262,42 @@ def scancode(ctx, input, output_file, copyright, license, package,
     Error and progress is printed to stderr.
     """
 
+    possible_scans = OrderedDict([
```

Comment:
I agree having a dedicated object instead of a tuple as the value in the dict would be more readable. However, it would probably not get rid of the `zip`ping as I do want to be able to access all first items of the tuple as a list to easily check them with `any`. Let me play around a bit with this...